pkg/asset/installconfig/azure/mock/azureclient_generated.go

```diff
@@ -105,8 +106,23 @@ func (m *MockAPI) GetResourcesProvider(ctx context.Context, resourceProviderName
 	return ret0, ret1
 }
 
-// GetResourcesProvider indicates an expected call of GetResourcesProvider.
+// GetResourcesProvider indicates an expected call of GetResourcesProvider
```

Comment:
I know this is automated, but it is strange that the periods are being removed from these GoDoc strings. Perhaps this is generated with an outdated version of gomock?

Reply:
Just installed latest gomock (1.4.3) and tried it. It still removed the period.

Reply:
I ran `$ sudo ./hack/go-genmock.sh ` against your PR and it is replacing the periods. `hack/go-genmock.sh` is the canonical way of generating mocks for the installer, so we don't want to commit anything that would deviate from that.

Can you try generating the mocks with `hack/go-genmock.sh` and let me know if you have problems?